### ISSUE TITLE
Scale problems. Does the addon auto-update? 

### ISSUE THREAD

**Issue by Vidyut** [makehumancommunity/mpfb2#128]
The addon was fine. All of a sudden it is creating giant humans that don't fit on the sets (like 10x). Everything else is the same, including presets used a hundred times to create the character reliably. Now my last desperate attempt to make a scifi promo has turned into Gulliver's travels...

**Comment by Vidyut** [comment]
Ignore. Seems to be working fine now. Something wonky was added by some plugin I used to make walls and a floor. I KNEW I should have used good old planes.

**Comment by Vidyut** [comment]
This self-resolved. I'm not sure what caused it, but I can no longer reproduce it.

**Comment by joepal1976** [comment]
For the protocol: No, MPFB will not do anything automatically.

However, it might be good to know that blender (ie blender as such, not MPFB) stores UI settings when you save a blend file. If you set scale to decimeter and save a first blend file, change it to meter and save a second blend file, then when you go back to the first blend file, the setting will be decimeter not meter.

**Comment by Vidyut** [comment]
I did assume that. But not content to assume, the scale was the first thing I checked to troubleshoot. The scale for all the interior files is in meters - all MPFB files work in this. exterior long shots (landscapish) is 100m. This was more like 10m - neither scales could have caused it. I strongly suspect it was Home Builder or some such - it scales walls, makes doors and so on and leaves a clutter of empties and what not. It may well be innocent in the matter, but the obsessive minimalist in me is giving it the suspicious eye anyway.
